### PR TITLE
v1.0.3

### PR DESCRIPTION
**Pull request by yotamloe**
- v1.0.3
  - Add support for opentelemetry detection
  - nodejs: check for existing `NODE_OPTIONS` 
  - python: remove metrics exporter

### CODE REVIEW

**Review by tamir-michaeli — APPROVED** [review]
Nice!

**Comment by yotamloe** [comment]
@tamir-michaeli I added tel detection to this pr, so I think we should review it again :)

File: detectors/opentelemetryDetector/inspectors/otel.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+			return true
+		}
+	}
+	return false
+
+}
+
+func otelInCmdLine(cmdLine string) bool {
+	return strings.Contains(cmdLine, otelStr) || strings.Contains(cmdLine, otlpStr)
+}
```
Comment (tamir-michaeli) [inline detectors/opentelemetryDetector/inspectors/otel.go:54]: For next versions, we need to make sure this does not clash with other otel libraries, i.e otel metrics agent etc.

**Review by tamir-michaeli — APPROVED** [review]
Amazing job! @yotamloe

**Pull request merged**